runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java

```diff
@@ -74,6 +75,36 @@ public static boolean browse(final String url)
 		return false;
 	}
 
+	/**
+	 * Tries to open a directory in the OS native file manager.
+	 * @param directory directory to open
+	 * @return @return true if operation was successful
+	 * THIS METHOD MAY BLOCK FOR A SUBSTANTIAL AMOUNT OF TIME.
```

Comment:
I would rather this just handle threading on it's own and not return anything; the return value is useless.